src/components/map/mapbox/Map.tsx

```diff
@@ -27,6 +27,16 @@ interface MapProps {
   ) => void;
 }
 
+const MapDefault = {
+  // Magic number, try and error by experience
+  DEBOUNCE_BEFORE_EVENT_FIRE: 1250,
+  CENTER_LONGITUDE: 147.3353554138993,
+  CENTER_LATITUDE: -42.88611707886841,
```

Comment:
this coordinate is the IMAS building

Reply:
Just a center point, with correct bbox it will cover most of the TAS area